packages/kit/types/index.d.ts

```diff
@@ -727,11 +727,13 @@ export function error(
 	body?: { message: string } extends App.Error ? App.Error | string | undefined : never
 ): HttpError;
 
+type HttpRedirectStatusCode = 300 | 301 | 302 | 303 | 304 | 305 | 306 | 307 | 308
+
 /**
  * Creates a `Redirect` object. If thrown during request handling, SvelteKit will
  * return a redirect response.
  */
-export function redirect(status: number, location: string): Redirect;
+export function redirect(status: HttpRedirectStatusCode, location: string): Redirect;
```

Comment:
Personally I would inline these, as the type is only used in one place (and I wouldn't want to export it for now).

Reply:
I knew this commend was coming ^^. Sure!